app-misc/pip3line/pip3line-9999.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Raw bytes manipulation, transformations (decoding and more) and interception"
+HOMEPAGE="https://github.com/metrodango/pip3line"
+if [[ ${PV} == 9999* ]] ; then
+inherit git-r3
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/metrodango/pip3line.git"
+EGIT_BRANCH="master"
+KEYWORDS=""
+else
+SRC_URI="https://github.com/metrodango/pip3line/archive/v${PV}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+IUSE="openssl qscintilla distorm python2_7 python3"
```

Comment:
Leaving Python2.7 aside, because it's unlikely to change in the near future, I would rather not link the plugin to a specific version of Python 3 by using the python target system.

The plugin loads natively the python3 library and does not care about which minor version as it uses base functionalities only.

I can change it if really required though, that's not really a problem.

Reply:
never mind just switched everything to using python_target